photon-client/src/test/scala/com/linkedin/photon/ml/estimators/GameEstimatorEvaluationFunctionTest.scala

```diff
@@ -43,6 +44,7 @@ class GameEstimatorEvaluationFunctionTest {
   private val random = new Random(1)
   private val regWeights = Array.fill[Double](4) { random.nextDouble }
   private val tolerance = MathConst.EPSILON
+  private val range = DoubleRange(0.001, 1000)
```

Comment:
Based on your conversations with Kinjal, this should actually be `DoubleRange(-3, 3)` - correct?

Reply:
Yes, you are right. More precisely in GAME model, the code used log to the base e. Here should be DoubleRange(log(0.001), log(1000)).